spec/factories/mentorships.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :mentorship do
+    mentor
```

Comment:
does this break existing tests?

Reply:
I don't believe so, but I'm noticing the 'focus' and think that I have maybe not checked